### DISCUSSION TITLE
torchcoder did not work after installation

### DISCUSSION THREAD

**Discussion by jdeawick** [community/community#75815] [Programming Help]
### Select Topic Area

Bug

### Body

I'm quite interested in the add-on to pyTorch, TorchCoder.  However, after installation (pip install), the basic code didn't work.
As a test, I tried to implement the example on the github page (https://github.com/hellojinwoo/TorchCoder).
It returned "QuickEncode" not found
I noticed that the script core.py on the github page wasn't installed.

Would love some insight; this package looks to be quite useful for intended applications

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @jdeawick  ! We're happy you're here, the Discussions category is solely related to conversations around the GitHub product Discussions.

If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://help.github.com/articles/adding-support-resources-to-your-project/), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://help.github.com/articles/setting-guidelines-for-repository-contributors/), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
